types/bootstrap/test/toast-tests.ts

```diff
@@ -10,6 +10,8 @@ new Toast(element, { animation: false });
 Toast.getInstance(element);
 // $ExpectType Toast
 Toast.getOrCreateInstance(element);
+// $ExpectType boolean
+Toast.isShown();
```

Comment:
isn't that one an instance method?
https://github.com/twbs/bootstrap/blob/main/js/src/toast.js#L133

Reply:
Yes, however I'm not sure how to express this as a test. In my code I'd use it like this:
```javascript
const myToast = new Toast(element, { animation: false });
const myBool = myToast.isShown(); 
```

Reply:
You can replace line 7 with your code.

Reply:
Thank you for your help @peterblazejewicz @martin-badin :)